### PR TITLE
Rewrite `events-listener` in Typescript

### PR DESCRIPTION
**Pull request by xhjkl**
This PR adds loose preliminary typings to `events-listener` service.

### CODE REVIEW

File: website/events-listener/package.json [hunk 1]
```diff
@@ -1,23 +1,24 @@
...
+  "version": "0.2.2",
   "description": "",
-  "main": "src/indes.js",
-  "type": "module",
+  "main": "src/indes.ts",
   "scripts": {
-    "start": "node dist/index.js",
-    "start:dev": "nodemon src/index.js",
-    "build": "cp -r src dist"
+    "start": "ts-node src/index.ts",
```
Comment (osipov-mit) [inline website/events-listener/package.json:7]: I think it's better to use ts-node in `start:dev` script and add here `build` script with running `tsc` command and use `node build/index.js` in start script.

File: website/events-listener/src/config.ts [hunk 2]
```diff
@@ -2,9 +2,9 @@ import dotenv from 'dotenv';
 import { strict as assert } from 'assert';
 dotenv.config();
 
-const checkEnv = (env) => {
+const checkEnv = (env: unknown): string => {
   assert.notStrictEqual(env, undefined);
-  return env;
+  return env as never;
```
Comment (osipov-mit) [inline website/events-listener/src/config.ts:7]: Why it's never?

**Review by osipov-mit — CHANGES_REQUESTED** [review]

File: website/events-listener/src/config.ts (hunk 2 above)
Comment (xhjkl) [reply website/events-listener/src/config.ts:7]: It's not really a never, just my my attempt to get away with casting to any ;3
Will think of better typings.

**Comment by osipov-mit** [comment]
Are you sure that without specifying `outDir: ./build` in tsconfig.json compiled code will be in `build` directory?

File: website/events-listener/src/config.ts (hunk 2 above)
Comment (osipov-mit) [reply website/events-listener/src/config.ts:7]: I guess `env` can only be a string

File: website/events-listener/package.json [hunk 3]
```diff
@@ -2,22 +2,26 @@
...
   "description": "",
-  "main": "src/indes.js",
+  "main": "src/index.ts",
   "type": "module",
   "scripts": {
-    "start": "node dist/index.js",
-    "start:dev": "nodemon src/index.js",
-    "build": "cp -r src dist"
+    "start": "node --experimental-specifier-resolution=node dist/index.js",
+    "start:dev": "nodemon src/index.ts",
```
Comment (osipov-mit) [inline website/events-listener/package.json:9]: You have to use ts-node with ts files. Nodemon will not work

**Review by osipov-mit — CHANGES_REQUESTED** [review]

File: website/events-listener/package.json (hunk 3 above)
Comment (osipov-mit) [reply website/events-listener/package.json:9]: waiting its resolving
@xhjkl

Comment (xhjkl) [reply website/events-listener/package.json:9]: I'm having another experience:
```
$ yarn start:dev
[nodemon] 2.0.15
[nodemon] to restart at any time, enter `rs`
[nodemon] watching path(s): *.*
[nodemon] watching extensions: ts,json
[nodemon] starting `ts-node src/index.ts`
08:14:18 INFO [Main] App is running...

```

Comment (osipov-mit) [reply website/events-listener/package.json:9]: Cool

**Review by osipov-mit — APPROVED** [review]

**Pull request merged**